search-enterprise/latest/using/unsupported.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Unsupported features
+---
+
+The following features, which were supported with Search and Insight Engine 2.x and Search Services 2.x (Solr) are not supported in the latest release for Search Enterprise 3.x (Elasticsearch).
+
+## Index and re-index
+
+* Indexing of nodes created during content repository bootstrap. For example, the sample site data. @Engineering what is this example?
+
+## Search features
+
+* Site queries
+* Path queries
+* Tag queries
+* Highlighting
+* Fingerprinting
+* Multilingual support (documents will be accepted and searchable with multiple languages, but only English grammar rules will be applied)
+* Template search
+* Resource limiting
+* Scoped search
+* Statistics
+* Search for an exact term
+* Fuzzy matching
+* Grouped search queries
+* Proximity search
+* Search for spans and positions
+* Search using date math
+* Search for boosts
+* Wildcards in phrase queries
+* Exact Match Queries
+* Field Facets Pagination
+* Field Facets Tags Exclusion
+
+## Field queries
+
+* PATH
+* PATHWITHREPEATS
+* PNAME
+* NPATH
+* PARENT
+* PRIMARYPARENT
+* QNAME
+* TAG
+* SITE
+* ANCESTOR
+* PRIMARYASSOCQNAME
+* PRIMARYASSOCTYPEQNAME
+* ISNULL
+* ISNOTNULL
+* ISUNSET
+* FINGERPRINT
+* ISROOT
+* ISCONTAINER
+* ASPECT
+* EXACTASPECT
+* CASCADETX
+* DBID
+* TX
+* TXID
+* INTXID
+* TXCOMMITTIME
+* ACLID
+* INACLTXID
+* ACLTXID
+* ACLTXCOMMITTIME
+* TENANT
+* OWNERSET
```

Comment:
To be added to the unsupported field list:

- READERSET
- DENYSET
- AUTHORITYSET
- id
- DBID
- FTSSTATUS

Reply:
Cheers @agazzarini  @mjstanford this contradicts your comment from the other day. You had said that READERSET doesnt need to go on here because it has never been supported in 2.0. Do you agree with what Andrea has said should it be added here? 
Might be worth verifying the other ones also. :-)

Reply:
I think I have done this correctly. I've added 

* Id
* DBID

As per comment from Martin S. Cheers :-)

Reply:
Copied over the bits from Angel that covers these comments.